src/main/java/org/sonar/plugins/csharp/CSharpCodeCoverageProvider.java

```diff
@@ -151,7 +151,7 @@ public CSharpIntegrationCoverageAggregator(Settings settings) {
   public static class CSharpIntegrationCoverageReportImportSensor extends CoverageReportImportSensor {
 
     public CSharpIntegrationCoverageReportImportSensor(CSharpIntegrationCoverageAggregator coverageAggregator, FileSystem fs) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "fs". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)